general/OKRs/README.md

```diff
@@ -0,0 +1,156 @@
+# Objectives and Key Results
+
+These are the company's OKR for 1Q2018.
+
+Status: Key Results need to be added
+
+Note: the fewer key results the better, the sample ones are just to illustrate where, **not** how many.
+
+### Vision:
+
+* Being the leader of AI on Code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Company Wide:
+
+* Having successful users of source{d}
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Being a better company to work at
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Product:
+
+* Launch of hosted playground
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Build a hosted MVP for open-source projects of source{d} PR* (ability to add applications that return results in PR's on GitHub)
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Launch of deduplication of code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Decide on the faith of legacy projects
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+\* we're lost on the name still, this objective needs its own document to explain better since it's a new product
+
+### Developer Community:
+
+* Create a network of influencers incl. Developers, Companies, Academia
+  * Create a forum (tbd whether it's slack, mailing list, or others) for influencers.
+  * Identify 20 influencers and invite them to join the community.
+  * Meet with all of the influencers that accept the invitation and welcome them personally.
+* Increase our presence in the developer community in Madrid & SF
+  * Organize 2 Go meetups in Madrid
```

Comment:
2 meetups in 3 months? Is is very archibable, is not expect te be archive at the 100%, the 70% should be fisable

Reply:
I could go with 3, since I really think that only two will take place haha